roles/uEsimulator/tasks/start.yml

```diff
@@ -0,0 +1,66 @@
+---
+
+- name: check if simluation is enabled else abort
+  fail:
+    msg: "Simulation is not enabled"
+  when: srsran.simulation == false
+
+- name: pull {{ srsran.docker.container.ue_image }} image
+  community.docker.docker_image:
+    name: "{{ srsran.docker.container.ue_image }}"
+    source: pull
+  when: inventory_hostname in groups['srsran_nodes']
+  become: true
+
+- name: remove /tmp/ue_zmq.conf
+  file:
+    path: "/tmp/ue_zmq.conf"
+    state: absent
+  when: inventory_hostname in groups['srsran_nodes']
+  become: true
+
+- name : copy ue_zmq config file to /tmp/ue_zmq.conf
+  template:
+    src: "{{ ROOT_DIR }}/{{ srsran.ue.conf_file }}"
+    dest: /tmp/ue_zmq.conf
+  when: inventory_hostname in groups['srsran_nodes']
+  become: true
+
+- name: create {{ srsran.docker.container.ue_image }} containers
+  community.docker.docker_container:
+    name: "rfsim5g-srsran-nr-ue"
+    image: "{{ srsran.docker.container.ue_image }}"
+    networks:
+      - name: "{{ srsran.docker.network.name }}"
+        # ipv4_address: "{{ srsran.ue.ip }}"
+    volumes:
+      - /tmp/ue_zmq.conf:/opt/ue_zmq.conf
+    devices:
+      - /dev/net/tun:/dev/net/tun
+    privileged: true
+    capabilities:
+    - NET_ADMIN
+    - NET_RAW
+    state: started
+    detach: true
+  when: inventory_hostname in groups['srsran_nodes']
+  become: true
+
+- pause:
+    seconds: 10
+
+#TODO: use ue_ip_pool for configuring default route for UE
```

Comment:
What need to be done here?

Reply:
need to remove hardcoded values for ip address. I have fixed it now.